sdcm/nemesis.py

```diff
@@ -3915,6 +3947,13 @@ def disrupt(self):
         self.disrupt_toggle_table_ics()
 
 
+class ToggleGcModeMonkey(Nemesis):
+    kubernetes = True
```

Comment:
let's be explicit about parameters (isn't this Nemesis disruptive=False?)

Reply:
yes, fixed.